app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/annual_exempt_amount.scala.html

```diff
@@ -76,10 +77,16 @@
             assistiveText = messages("unit-input.assistive.pounds")
         )
 
-        <p>@Html(messages(s"$key.link", appConfig.annualExemptAmountUrl))</p>
+        @if(isTrust) {
+          @details(messages(s"$key.details.1.header"), Html(messages(s"$key.details.1.body", appConfig.taxFreeAllowance)))
+          @details(messages(s"$key.details.2.header"),  Html(messages(s"$key.details.2.body", appConfig.trustsForVulnerable)))
+        } else {
+          <p>@Html(messages(s"$key.link", appConfig.annualExemptAmountUrl))</p>
+        }
 
         @submitButton(messages("button.saveAndContinue"))
     }
 
     @returnToSummaryLink(displayReturnToSummaryLink = true)
 }
+
```

Comment:
the help text needs to change for trusts as well

Reply:
It does at the moment. Maybe it's not very clear, happy to change it. But the help text message key gets read as `annualExemptAmount.individual.helpText` or annualExemptAmount.trust.helpText` on line 65